react/week4/user_search/src/App.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
+import Login from "./Login";
```

Comment:
Maybe Search is a better name than Login? 

The common approach is to make a component folder and put js files for components there ie
import Search from '../components/Search';